controller/system_backup_controller.go

```diff
@@ -362,6 +366,15 @@ func (c *SystemBackupController) reconcile(name string, backupTargetClient engin
 		return
 	}
 
+	// If the system backup is in the final state, we don't need to do anything for the backup target not found.
+	if backupTarget == nil && !systemBackupInFinalState(systemBackup) {
+		c.updateSystemBackupRecord(record,
+			systemBackupRecordTypeError, longhorn.SystemBackupStateError,
+			constant.EventReasonFailedStarting, SystemBackupErrBackupTarget,
+		)
+		return fmt.Errorf("default backup target not found")
```

Comment:
Can just reuse the const message, `SystemBackupErrDefaultBackupTarget`.

Reply:
Reused. Thanks.